src/markdown-pages/docs/ecosystem/ecosystem-role-based-access.md

```diff
@@ -0,0 +1,208 @@
+| admin     | A Galasa service administrator. Able to do any operation supported by the Galasa service | all |
+| owner       | The owner of the Galasa service. Such users have all the rights of an administrator, but nobody can delete their id or change their role without changing the kubernetes configuration | all |
+
+
+
+## Constraints
+The system also has some constraints on how these resources can be set up and used:
+- Nobody can change their own "role", assigning a different role to yourself is not permitted
+- New users on the system are assigned a default role when they first login to the Galasa service. This role is configurable. More details on how to configure this default role is described in section [how to set the default user role](#how-to-set-the-default-user-role).
+- One or more users can be assigned the `owner` role. Such users are privileged because an administrator role cannot directly update their role away from `owner` using the standard mechanisms. Similarly, such "owners" of the Galasa service cannot be removed, even by another "owner" without changing kubernetes configuration.
+- An administrator cannot delete their own user record. This constraint attempts to make sure that there is at least one owner or administrator on the system. If nobody active in your organisation has `admin` or `owner` rights, then a new owner of the service can be nominated using the Galasa service install/update instructions. See [How to nominate a user as the Galasa service `owner`](#how-to-nominate-a-user-as-the-galasa-service-owner)
+- An administrator cannot change their own role.
+- Not even administrators can change the role of a user nominated as an `owner`.
+- The `owner` role can only be assigned using the Kubernetes property, not via the command-line or REST interface.
+
+## Upgrading to a version of Galasa which supports RBAC
+When upgrading your Galasa service to a version 0.40.0 or later, any existing users will be assigned the role of `admin` without any extra action being required.
+
+Under such circumstances you may wish to assign the role of `tester` to anyone who now should not have administration rights.
+
+Upgrading the existing Galasa service will therefore not require any `owner` to be set up, because your system will already have multiple administrator users with the `admin` role.
+
+
+## Setting up RBAC when you install Galasa into Kubernetes
+
+### How to nominate a user as the Galasa service `owner`
+An "owner" of the Galasa service is a user who has been nominated as the owner of the service, and who has the `owner` RBAC role.
+
+This nomination is made at the time the Galasa service is first installed into the Kubernetes namespace, or 
+when it is subsequently updated using the Galasa Helm chart.
+
+To nominate a user as an `owner` you need to set the property in the Helm values file when the Galasa service is installed or updated.
+
+For example, in the `.values` file used by helm:
```

Comment:
"values.yaml" instead of ".values" and "Helm" instead of "helm"?

Reply:
Fixed.